saleor/order/actions.py

```diff
@@ -421,7 +421,7 @@ def fulfillment_tracking_updated(
     events.fulfillment_tracking_updated_event(
         order=fulfillment.order,
         user=user,
-        app=app,
+        app=app if app and app.removed_at is not None else None,
```

Comment:
Is this expected on 3.17 only? I don't see this change in 3.14 or older version ports.

Reply:
this line is redundant since the app comes from `get_app_promise` which is already updated to the new flow. I will remove this from all affected PR's